source/vocab/relations.ttl

```diff
@@ -594,6 +616,14 @@
     ptg:abstract true;
     owl:equivalentProperty bf2:contributionOf .
 
+:contributor a owl:DatatypeProperty;
+    :category :shorthand;
+    rdfs:label "Contributor"@en, "Medverkande"@sv;
+    rdfs:domain :Document; # TODO?
```

Comment:
```suggestion
    sdo:domainIncludes :Endeavour ;
```
Same as for `:contribution` (which admittedly isn't ideal, but works for now).

Reply:
But then it doesn't work on `ImageObject`?

Reply:
```suggestion
    rdfs:domain :Resource;
```
We want to be able to add an image on any Resource?

Reply:
```suggestion
    sdo:domainIncludes :Endeavour, :MediaObject ;
```